ci_framework/roles/edpm_build_images/tasks/post.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.  
+
+- name: Push edpm-hardened-uefi image to registry
+  containers.podman.podman_image:
+    name: "{{ item }}"
+    push_args:
```

Comment:
2023-07-04 13:08:57.323522 | controller | TASK [edpm_build_images : Push edpm-hardened-uefi image to registry name={{ item }}, push_args={'dest': '{{ push_registry }}/{{ registry_namespace }}'}, tag=sandeep-testing, push=True] ***
2023-07-04 13:08:57.323531 | controller | Tuesday 04 July 2023  13:08:57 -0400 (0:00:11.824)       0:14:44.309 **********
2023-07-04 13:08:58.488255 | controller | failed: [localhost] (item=edpm-hardened-uefi) => {"ansible_loop_var": "item", "changed": false, "item": "edpm-hardened-uefi", "msg": "Failed to pull image edpm-hardened-uefi:sandeep-testing"}
2023-07-04 13:08:58.488313 | controller | failed: [localhost] (item=ironic-python-agent) => {"ansible_loop_var": "item", "changed": false, "item": "ironic-python-agent", "msg": "Failed to pull image ironic-python-agent:sandeep-testing"}

We need to add `pull: false`